pkg/api/vlabs/types.go

```diff
@@ -118,6 +118,14 @@ const (
 // OrchestratorProfile contains Orchestrator properties
 type OrchestratorProfile struct {
 	OrchestratorType OrchestratorType `json:"orchestratorType"`
+	KubernetesConfig KubernetesConfig `json:"kubernetesConfig,omitempty"`
+}
+
+// KubernetesConfig contains the Kubernetes config structure, containing
+// Kubernetes specific configuration
+type KubernetesConfig struct {
```

Comment:
@wangtt03 your change can stay as is. We will look into introducing OrchestratorOptions at some point in the future but the current setup is extensible so we should be good.